### PR TITLE
Add back our contributing guide

### PR DESCRIPTION
**Pull request by jamiehannaford**
Helps newcomers get involved in the project, etc.

/cc @ss22ever

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+* how easy was it to start using our SDK?
+* did it meet your expectations? If not, why not?
+* did our documentation help or hinder you?
+* what could we improve in general?
+
+### 2. Fixing bugs
+
+If you want to start fixing open bugs, we'd really appreciate that! Bug fixing
+is central to any project. The best way to get started is by heading to our
+[bug tracker](https://github.com/rackspace/gophercloud/issues) and finding open
```
Comment (ss22ever) [inline CONTRIBUTING.md]: @jamiehannaford will it be https://github.com/gophercloud/gophercloud/issues?

Comment (jamiehannaford) [reply CONTRIBUTING.md]: yep!

Comment (ss22ever) [reply CONTRIBUTING.md]: Can I submit a pr?

Comment (jamiehannaford) [reply CONTRIBUTING.md]: sure, go for it

Comment (jamiehannaford) [reply CONTRIBUTING.md]: after your post, I'll close this one so it can be superseded by yours

**Comment by jrperritt** [comment]
It's [here](https://github.com/gophercloud/gophercloud/blob/master/.github/CONTRIBUTING.md) and linked in the README [here](https://github.com/gophercloud/gophercloud#contributing)

**Comment by jamiehannaford** [comment]
Ah my bad! Sorry @jrperritt 

@ss22ever If you need to add anything, please do it to that file

**Pull request closed without merging**